test/components/joiners/test_document_joiner.py

```diff
@@ -145,6 +145,36 @@ def test_run_with_distribution_based_rank_fusion_join_mode(self):
         ]
         assert all(doc.id in expected_document_ids for doc in output["documents"])
 
+    def test_run_with_distribution_based_rank_fusion_join_mode(self):
```

Comment:
Duplicate test name. I'd add a prefix to indicate that we're testing for zero-scores.